alicloud/resource_alicloud_alikafka_instance_test.go

```diff
@@ -216,6 +267,50 @@ func TestAccAlicloudAlikafkaInstance_basic(t *testing.T) {
 
 }
 
+func testAccCheckErr(t *testing.T, err error) {
+
+	if err != nil {
+		t.Skipf(fmt.Sprintf("skip testing, error occur: %s", err))
+		t.Skipped()
+	}
+}
+
+func getVSwitch() (string, error) {
```

Comment:
The getVSwitch can be replaced by:
```
data "alicloud_vswitches" "default" {
  is_default = true
}
```
and the line https://github.com/terraform-providers/terraform-provider-alicloud/pull/1873/files#diff-70213675db9bca8b6c45d3ffa32e5c79R122 can be replaced by `"vswitch_id":  "${data.alicloud_vswitches.default.ids.0}"`